lib/queryBuilder/transformations/WrapMysqlModifySubqueryTransformation.js

```diff
@@ -2,6 +2,8 @@
 
 const { QueryTransformation } = require('./QueryTransformation');
 const { isMySql } = require('../../utils/knexUtils');
+const { once } = require('../../utils/objectUtils');
+const getJoinBuilder = once(() => require('../JoinBuilder').JoinBuilder);
```

Comment:
isn't it a slight over-optimization? Wouldn't this suffice? :)
```suggestion
const { JoinBuilder } = require('../JoinBuilder');
```

Reply:
ok it seems it is necessary - same logic is in
https://github.com/ovos/objection.js/blob/412bb77b9cf69496e477c38fc10bca679ceca461/lib/queryBuilder/operations/ObjectionToKnexConvertingOperation.js#L7